py/desispec/scripts/specex.py

```diff
@@ -454,11 +455,12 @@ def mean_psf(inputs, output):
                 log.info("for fiber bundle {}, {} valid PSFs".format(bundle,
                     ok.size))
 
-            if ok.size>=2 : # use median
-                log.debug("bundle #{} : use median".format(bundle))
+            if ok.size>=2 : # use mean
+                log.debug("bundle #{} : use mean".format(bundle))
                 for f in fibers_in_bundle[bundle]  :
-                    output_coeff[f]=np.median(coeff[ok,f],axis=0)
-                output_rchi2[bundle]=np.median(bundle_rchi2[ok,bundle])
+                    output_coeff[f]=np.mean(coeff[ok,f],axis=0)
+                output_rchi2[bundle]=np.mean(bundle_rchi2[ok,bundle])
```

Comment:
This is another case of a change in the default behavior.  How extensively have you tested the robustness of this median -> mean change?  Ditto for x/ytrace median -> mean below.  Should we have a sigma-clipped mean instead?

Reply:
added explanation in code:
            # We finally resorted to use a mean instead of a median here for two reasons.
            # First, there is already a vetting of PSF bundles with good chi2 above
            # that protects us from bad fits (we only expect outliers because of bad fits because of cosmic rays,
            # not a glitch in hardware). Second, some of the PSF parameters have large correlations,
            # which mean that two pairs of parameter values, like (p_a_i,p_b_i) and (p_a_j,p_b_j) (with a,b param
            # indexes and i,j exposure indices) may give similar PSFs despite large noise in individual parameters
            # but a median could decide to select a pair like (p_a_i,p_b_j) that could lead to a PSF inconsistent
            # with data. Using a mean instead of a median protects us from this situation.